src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -29,7 +35,7 @@ export type Props<L> = PropsWithSpread<
     /**
      * The menu content (if the links prop is not supplied).
      */
-    children?: ReactNode;
+    children?: ReactNode | ((close: () => void) => ReactElement);
```

Comment:
This could be `ContextualMenuDropdownProps["children"]` to save some duplication.